src/safeds/data/tabular/operator/_temporal.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from safeds.data.tabular.containers import Column
+
+
+class Temporal:
```

Comment:
Call this `TemporalCell`, the file `_temporal_cell` and move it next to `_string_cell`: https://github.com/Safe-DS/Library/blob/69a780cab1a5dcab33c8c7a36f03bd2eb3367683/src/safeds/data/tabular/containers/_string_cell.py#L12

Should operate on cells, not columns, like `StringCell`.